src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -94,6 +100,45 @@ const ClaimsRow: React.FC<ClaimsRowData> = ({ claim, group, children }) => {
   );
 };
 
+const DeletedUserClaimsRow: React.FC<ClaimsRowData> = ({ group, children }) => {
+  return (
+    <tr>
+      <td>
+        <Flex>
+          <Text>Unknown</Text>
+        </Flex>
+      </td>
+      <td>
+        <Flex css={{ gap: '$sm' }}>
+          <Text>Unknown</Text>
+        </Flex>
+      </td>
```

Comment:
Not sure if this is what we want for the org and circle name for deleted users. Wondering if we should just leave these blank